data/Hints/coop.json

```diff
@@ -64,8 +64,8 @@
         "dual_always":      {"order": 3, "weight": 0.0, "fixed":   0, "copies": 2},
         "always":           {"order": 4, "weight": 0.0, "fixed":   0, "copies": 2},
         "named-item":       {"order": 5, "weight": 0.0, "fixed":   1, "copies": 1},
-        "sometimes":        {"order": 6, "weight": 0.0, "fixed":   6, "copies": 2},
-        "dual":             {"order": 7, "weight": 0.0, "fixed":   1, "copies": 2},
+        "sometimes":        {"order": 7, "weight": 0.0, "fixed": 100, "copies": 2},
+        "dual":             {"order": 6, "weight": 0.0, "fixed":   1, "copies": 2},
```

Comment:
could you swap these two lines so the definition order matches the `order` field?